.eslintrc.js

```diff
@@ -191,5 +191,11 @@ module.exports = {
         'react/prefer-stateless-function': 'off',
       },
     },
+    {
+      files: ['scripts/**/*.{ts,tsx,js}'], // Modify the pattern to match your specific directory
```

Comment:
Can we change this comment to be more specific about why we need this override? That will help future readers understand if it's still relevant or needed.